src/Widgets/HeaderBar.vala

```diff
@@ -199,45 +197,45 @@ namespace Quilter.Widgets {
             color_button_dark_context.add_class ("color-dark");
 
             color_button_dark.clicked.connect (() => {
-                settings.dark_mode = true;
-                settings.sepia_mode = false;
-                settings.moon_mode = false;
+                gsettings.set_boolean("dark-mode", true);
+                gsettings.set_boolean("sepia-mode", false);
+                gsettings.set_boolean("moon-mode", false);
```

Comment:
Yeah this is where it really shows how this should be an enum and not three separate settings

Reply:
:stuck_out_tongue_winking_eye: